gensim/corpora/bleicorpus.py

```diff
@@ -28,24 +28,22 @@ class BleiCorpus(IndexedCorpus):
 
     Each document is one line::
 
-      N fieldId1:fieldValue1 fieldId2:fieldValue2 ... fieldIdN:fieldValueN
+        N fieldId1:fieldValue1 fieldId2:fieldValue2 ... fieldIdN:fieldValueN
 
-    The vocabulary is a file with words, one word per line; word at line K has an
-    implicit ``id=K``.
+
+    The vocabulary is a file with words, one word per line; word at line K has an implicit `id=K`.
 
     """
 
     def __init__(self, fname, fname_vocab=None):
         """
-        Initialize the corpus from a file.
 
         Parameters
         ----------
         fname : str
-            Serialized corpus's filename.
-        fname_vocab : str or None, optional
-            Vocabulary file. If not given, searching for the
-            vocab/vocab.txt file.
+            File path to Serialized corpus.
+        fname_vocab : str, optional
+            Vocabulary file. If `fname_vocab` is None, searching for the vocab.txt or `fname_vocab`.vocab file.
```

Comment:
Are you sure it's `fname_vocab`.vocab? `fname_vocab` is none, isn't it?

Reply:
Yep

Reply:
Not quite, I added correct description

Reply:
Still don't get it. It should be \`fname\`.vocab, \`fname_vocab\` is undefined!

Reply:
Not quite :) I go through the code with ipdb for this case, this is significantly "wider" that we discuss here (I already fix it).